clang/test/SemaSYCL/address-space-conversions.cpp

```diff
@@ -0,0 +1,75 @@
+// RUN: %clang_cc1 -fsycl-is-device -verify -fsyntax-only %s
```

Comment:
Please add a comment here about what the test tests.

Reply:
Done.